src/02/time.asm

```diff
@@ -1,3 +1,15 @@
+;; clockSupported [Time]
+;;   Returns whether the clock is supported.
+;; Outputs:
+;;   Z: set when the clock is supported; reset otherwise
+clockSupported:
+#ifdef CLOCK
+    cp a ; set Z
+#else
+    or 1 ; reset Z
```

Comment:
If unsupported, load A with `errUnsupported`, which is how other similar functions do it.